### ISSUE TITLE
Feature request: Dark mode

### ISSUE THREAD

**Issue by hakunamatatarr** [mb1986/rm-hacks#307]
It would be cool to have a dark mode toggle in the document settings that inverts the definition of black and white!

**Comment by PepikVaio** [comment]
Believe me, it wasn't. Did you have an older version of FW? The settings were in dark mode and it wasn't good. 
In general, it seems to me that e-ink isn't built for that.

**Comment by hakunamatatarr** [comment]
I just bought the remarkable recently. But following your comment, I searched a bit more and saw that there are still black templates hidden in the software that can be activated. I did that and I agree that there is much more ghosting but otherwise I really like it. However, using it this way has some compatibility problems of copy/pasting black writing etc. 
**TL;DR** I would still like a bright/dark mode switch if it is not a big hustle to implement.

**Comment by sqrpshr13** [comment]
It's mostly a feature we want for streaming presentations.
